src/core/evaluator.js

```diff
@@ -2269,21 +2277,26 @@ class PartialEvaluator {
 
             break;
           case OPS.beginMarkedContent:
+            markedContentLevel++;
+            if (args?.some(arg => arg instanceof Dict)) {
+              warn(`getOperatorList - ignoring operator: ${fn}`);
+              continue;
+            }
+            break;
           case OPS.endMarkedContent:
+            markedContentLevel--;
```

Comment:
You need to protect against this becoming negative, note https://github.com/mozilla/pdf.js/blob/b7b8e5ef788313451e869c2816a87f943bba35c3/src/core/evaluator.js#L3426-L3434